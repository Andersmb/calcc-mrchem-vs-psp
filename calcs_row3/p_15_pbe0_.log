Starting job 5688127 on c1-[21,26],c3-[13,27],c5-22,c10-5 at Fri May 20 04:05:02 CEST 2022

Copying p_15_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying p_15_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688127      p_15_pbe0_        180                                             00:30:49      0:0 
5688127.bat+      batch         24        1   00:00:03          0   00:00:03   00:30:49      0:0 
5688127.ext+     extern        180        6   00:00:00          5   00:00:00   00:30:49      0:0 
5688127.0      mrchem.x        180       30   00:30:12         29   01:17:55   00:30:25      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688127                                                                          
5688127.bat+     19336K          0     19336K        0              0          0 
5688127.ext+          0          5          0        0              5          0 
5688127.0     11873732K         29 5451876898        0             15          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688127                                                                                               
5688127.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688127.ext+        0.00M               5          0.00M            0                5              0 
5688127.0           7.32M              28          6.83M      117.97M               12         56.89M 

Job 5688127 completed at Fri May 20 04:35:52 CEST 2022
