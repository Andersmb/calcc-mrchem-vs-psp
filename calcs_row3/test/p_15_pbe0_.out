                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :       -0.139697        0.794715       -0.668027
    1       H            :       -2.807987        0.794715       -0.668027
    2       F            :        0.188363        0.794715        2.303463
    3       F            :        0.188363       -2.132525       -1.178917
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              17
 Beta electrons          :                                              17
 Total electrons         :                                              34
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000206882e+00
    1    1    a          :                              9.999999010593e-01
    2    1    a          :                              9.999999995764e-01
    3    1    a          :                              1.000000008432e+00
    4    1    a          :                              1.000000013204e+00
    5    1    a          :                              1.000000043608e+00
    6    1    a          :                              1.000000001235e+00
    7    1    a          :                              9.999999618955e-01
    8    1    a          :                              9.999999804376e-01
    9    1    a          :                              9.999994849955e-01
   10    1    a          :                              9.999999629096e-01
   11    1    a          :                              1.000000144428e+00
   12    1    a          :                              9.999999828005e-01
   13    1    a          :                              9.999997017561e-01
   14    1    a          :                              9.999973287572e-01
   15    1    a          :                              9.999999636935e-01
   16    1    a          :                              9.999999747213e-01
   17    1    b          :                              1.000000018726e+00
   18    1    b          :                              9.999992329143e-01
   19    1    b          :                              9.999999237253e-01
   20    1    b          :                              1.000000023198e+00
   21    1    b          :                              1.000000013199e+00
   22    1    b          :                              1.000000043607e+00
   23    1    b          :                              1.000000001235e+00
   24    1    b          :                              9.999999625613e-01
   25    1    b          :                              9.999999807794e-01
   26    1    b          :                              9.999999419978e-01
   27    1    b          :                              9.999998239174e-01
   28    1    b          :                              1.000000070086e+00
   29    1    b          :                              9.999999802777e-01
   30    1    b          :                              9.999999851965e-01
   31    1    b          :                              9.999999195692e-01
   32    1    b          :                              9.999999551756e-01
   33    1    b          :                              1.000000304839e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                536.637896132030
 E-N energy              :            (au)              -1519.382746410868
 Coulomb energy          :            (au)                370.682971929658
 Exchange energy         :            (au)                -10.816474997794
 X-C energy              :            (au)                -33.551313009104
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                118.006953997490
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -656.429666356078
 Nuclear energy          :            (au)                118.006953997490
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.384227123586e+02
                         :      (kcal/mol)             -3.378653503297e+05
                         :        (kJ/mol)             -1.413628626697e+06
                         :            (eV)             -1.465122774565e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -76.908915292571
    1    1    a          :            (au)                -24.492989319088
    2    1    a          :            (au)                -24.491096448102
    3    1    a          :            (au)                 -6.762353174698
    4    1    a          :            (au)                 -4.903634896850
    5    1    a          :            (au)                 -4.901002866180
    6    1    a          :            (au)                 -4.893630915064
    7    1    a          :            (au)                 -1.217187672595
    8    1    a          :            (au)                 -1.168516224667
    9    1    a          :            (au)                 -0.667225586953
   10    1    a          :            (au)                 -0.498151573973
   11    1    a          :            (au)                 -0.462576543880
   12    1    a          :            (au)                 -0.441664779890
   13    1    a          :            (au)                 -0.362052887546
   14    1    a          :            (au)                 -0.359304739530
   15    1    a          :            (au)                 -0.352976703116
   16    1    a          :            (au)                 -0.272179612080
   17    1    b          :            (au)                -76.908926991745
   18    1    b          :            (au)                -24.492980652334
   19    1    b          :            (au)                -24.490250450587
   20    1    b          :            (au)                 -6.762352955697
   21    1    b          :            (au)                 -4.903634130186
   22    1    b          :            (au)                 -4.901001650500
   23    1    b          :            (au)                 -4.893629650078
   24    1    b          :            (au)                 -1.217166914191
   25    1    b          :            (au)                 -1.168536281975
   26    1    b          :            (au)                 -0.667218325152
   27    1    b          :            (au)                 -0.498148953470
   28    1    b          :            (au)                 -0.462574270293
   29    1    b          :            (au)                 -0.441663384446
   30    1    b          :            (au)                 -0.362036069716
   31    1    b          :            (au)                 -0.359305965378
   32    1    b          :            (au)                 -0.352978274010
   33    1    b          :            (au)                 -0.272180938981
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -306.310045095520
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.830952e+00        -538.422712358587         -5.384227e+02
    1          3.624631e-01        -541.346146128258         -2.923434e+00
    2          1.506639e-01        -541.461715300429         -1.155692e-01
    3          9.269496e-02        -541.482913449313         -2.119815e-02
    4          4.880405e-02        -541.489926017499         -7.012568e-03
    5          5.134018e-02        -541.494990663939         -5.064646e-03
    6          1.537481e-02        -541.495546373575         -5.557096e-04
    7          1.242888e-02        -541.495873820105         -3.274465e-04
    8          4.327461e-03        -541.495915020564         -4.120046e-05
    9          3.156665e-03        -541.495930083595         -1.506303e-05
   10          1.265239e-03        -541.495933353524         -3.269929e-06
   11          9.494604e-04        -541.495935033309         -1.679784e-06
   12          5.006438e-04        -541.495935462627         -4.293184e-07
   13          2.292189e-04        -541.495935578689         -1.160618e-07
   14          1.584644e-04        -541.495935619793         -4.110393e-08
   15          1.326005e-04        -541.495935642525         -2.273202e-08
---------------------------------------------------------------------------
                      SCF converged in 15 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :       -0.139697        0.794715       -0.668027
    1       H            :       -2.807987        0.794715       -0.668027
    2       F            :        0.188363        0.794715        2.303463
    3       F            :        0.188363       -2.132525       -1.178917
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                540.466033199977
 E-N energy              :            (au)              -1526.407469492435
 Coulomb energy          :            (au)                370.738013639346
 Exchange energy         :            (au)                -10.788462303182
 X-C energy              :            (au)                -33.511004683722
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                118.006953997490
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -659.502889640015
 Nuclear energy          :            (au)                118.006953997490
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.414959356425e+02
                         :      (kcal/mol)             -3.397938270407e+05
                         :        (kJ/mol)             -1.421697373261e+06
                         :            (eV)             -1.473485440778e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.312086404563
    1    1    a          :            (au)                -24.797534724407
    2    1    a          :            (au)                -24.797530320189
    3    1    a          :            (au)                 -6.701058070910
    4    1    a          :            (au)                 -4.839369996868
    5    1    a          :            (au)                 -4.837099645563
    6    1    a          :            (au)                 -4.831100587957
    7    1    a          :            (au)                 -1.283105193267
    8    1    a          :            (au)                 -1.250833155896
    9    1    a          :            (au)                 -0.686636285500
   10    1    a          :            (au)                 -0.560863322751
   11    1    a          :            (au)                 -0.544542271244
   12    1    a          :            (au)                 -0.520146482333
   13    1    a          :            (au)                 -0.478064969942
   14    1    a          :            (au)                 -0.459839395190
   15    1    a          :            (au)                 -0.446389382960
   16    1    a          :            (au)                 -0.294381304248
   17    1    b          :            (au)                -77.312086415700
   18    1    b          :            (au)                -24.797534869055
   19    1    b          :            (au)                -24.797530180270
   20    1    b          :            (au)                 -6.701058062878
   21    1    b          :            (au)                 -4.839370028422
   22    1    b          :            (au)                 -4.837099706914
   23    1    b          :            (au)                 -4.831100583463
   24    1    b          :            (au)                 -1.283105216105
   25    1    b          :            (au)                 -1.250833174331
   26    1    b          :            (au)                 -0.686636379713
   27    1    b          :            (au)                 -0.560863382447
   28    1    b          :            (au)                 -0.544542287266
   29    1    b          :            (au)                 -0.520146518677
   30    1    b          :            (au)                 -0.478064986496
   31    1    b          :            (au)                 -0.459839383796
   32    1    b          :            (au)                 -0.446389333845
   33    1    b          :            (au)                 -0.294381403802
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -309.281163426969
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        1.174125       -0.392233        0.329652
 Magnitude               :            (au)                        1.281049
                         :         (Debye)                        3.256101
---------------------------------------------------------------------------
 Nuclear vector          :       -1.512902        0.675164       -0.567534
 Magnitude               :            (au)                        1.751232
                         :         (Debye)                        4.451188
---------------------------------------------------------------------------
 Total vector            :       -0.338778        0.282931       -0.237882
 Magnitude               :            (au)                        0.501406
                         :         (Debye)                        1.274447
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 30m 13s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
